.papr.yml

```diff
@@ -3,34 +3,44 @@ branches:
     - auto
     - try
 
+context: f27-codestyle
+required: true
+container:
+  image: registry.fedoraproject.org/fedora:28
+
+tests:
+  - ci/ci-commitmessage-submodules.sh
+  - ci/codestyle.sh
+
+---
+
 context: f27-primary
+inherit: true
 
-required: true
+# FIXME; temporary workaround
+# https://github.com/ostreedev/ostree/pull/1513#issuecomment-378784162
+host:
+  distro: fedora/27/atomic
+  specs:
+    ram: 4096
+#container:
+#  image: registry.fedoraproject.org/fedora:27
 
-cluster:
-  hosts:
-    - name: vmcheck1
-      distro: fedora/27/atomic
-    - name: vmcheck2
-      distro: fedora/27/atomic
-    - name: vmcheck3
-      distro: fedora/27/atomic
-  container:
-    image: registry.fedoraproject.org/fedora:27
+tests:
+  - cd /etc/yum.repos.d/ && curl -L -O https://copr.fedorainfracloud.org/coprs/walters/oci-kvm-hook/repo/fedora-27/walters-oci-kvm-hook-fedora-27.repo
+  - rpm-ostree install oci-kvm-hook && rpm-ostree ex livefs
```

Comment:
If we're passing `/dev/kvm` anyway, then there's no point in installing `oci-kvm-hook`, right?